frontend/taipy-gui/base/src/app.ts

```diff
@@ -101,6 +104,16 @@ export class TaipyApp {
         this._onReload = handler;
     }
 
+    get onWsMessage() {
```

Comment:
I thought you would only defined the setter and let the on<whatever> (ie onWsMessage here) be a real method with parameter that would check the existence of this._onWsMessage and call it

ie 
```
onWsMessage = (event, payload) => {
  this._onWsMessage && this._onWsMessage(event, payload);
}
```
I think we're loosing params checking with a single onUserEvent

Reply:
since onWsMessage namespace was occupied by the set, you could not actually have a normal function. It is reserved already by the set method and I tried to remove the get function and definied a normal function and ts screamed at me :D